### PR TITLE
[Console-IoT]-Add ID to IoT project detail

### PR DESCRIPTION
**Pull request by suyash-naithani**
### Type of change

- Refactoring

### Description

Add ID to IoT project detail module

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/modules/iot-project-detail/components/DeviceRegistrationManagement/DeviceRegistrationManagement.tsx [hunk 1]
```diff
@@ -235,7 +236,7 @@ const DeviceRegistationManagement: React.FunctionComponent<IDeviceRegistationMan
                 value={endpoiuntUrl}
                 isReadOnly={true}
                 enableCopy={true}
-                id={"drm-input-endpoiunt-url"}
+                id={"device-reg-endpoiunt-url-input"}
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/iot-project-detail/components/DeviceRegistrationManagement/DeviceRegistrationManagement.tsx:239]: Typo 'endpoint'

File: console/console-init/ui/src/modules/iot-project-detail/components/GeneralInfo/GeneralInfo.tsx [hunk 2]
```diff
@@ -106,6 +115,8 @@ const GeneralInfo: React.FunctionComponent<IGeneralInfoProps> = ({
           <b className={css(styles.style_margin)}>Telemetry address name</b>
           {telemetryAddress && (
             <Button
+              id="general-info-navigate-telemetry-address-button"
+              aria-label="Navigate to telemetry-address button"
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/modules/iot-project-detail/components/GeneralInfo/GeneralInfo.tsx:119]: This aria-label is not making much sense. we can omit button from the sentence.

File: console/console-init/ui/src/modules/iot-project-detail/components/AccessCredentials/AccessCredentials.tsx [hunk 3]
```diff
@@ -239,7 +244,10 @@ const AccessCredentials: React.FunctionComponent<IAccessCredentialsProps> = ({
           <br />
           <Messaging />
           <br />
-          <AdapterList id="ac-adapter-list-container" adapters={adapters} />
+          <AdapterList
+            id="access-credentials-adapter-list-container"
```
Comment (obabec) [inline console/console-init/ui/src/modules/iot-project-detail/components/AccessCredentials/AccessCredentials.tsx:248]: I would remove `container` from ID

File: console/console-init/ui/src/modules/iot-project-detail/components/AccessCredentials/AccessCredentials.tsx [hunk 4]
```diff
@@ -54,7 +54,8 @@ const AccessCredentials: React.FunctionComponent<IAccessCredentialsProps> = ({
   };
   const EditIcon = () => (
     <SwitchWithToggle
-      id="access-credential-switch"
+      id="access-credentials-view-json-switchtoggle"
```
Comment (ajaypratap003) [inline console/console-init/ui/src/modules/iot-project-detail/components/AccessCredentials/AccessCredentials.tsx:57]: i would like to suggest small change in this id.
 `id="access-credentials-view-json-switch-button"`

**Pull request merged**